neet/boolean/sensitivity.py

```diff
@@ -489,12 +489,18 @@ def average_sensitivity(self, states=None, weights=None, calc_trans=True):
 
         return np.sum(Q) / self.size
 
-    def C_sensitivity_at_x(self, state, transitions=None, c=1):
+    def c_sensitivity(self, state, transitions=None, c=1):
+
+        assert !(c < 0),"the value of c needs to be greater than or equal to zero"
```

Comment:
Paren problem
```python
assert (!(c > 0)), "the value of c needs to be greater than or equal to zero"
```
or even better
```suggestion
        assert(c >= 0), "the value of c needs to be greater than or equal to zero"
```
This will fix the failing tests. (See the line below as well.)